pages/c/_cluster/apps/charts/index.vue

```diff
@@ -42,21 +84,64 @@ export default {
 
   data() {
     return {
-      allRepos:            null,
-      category:            null,
-      operatingSystem:     null,
-      searchQuery:         null,
-      showDeprecated:      null,
-      showHidden:          null,
+      allRepos:             null,
+      category:             null,
+      operatingSystem:      null,
+      searchQuery:          null,
+      showDeprecated:       null,
+      showHidden:           null,
+      sliders:         sliderData,
+      chartMode:            'featured',
```

Comment:
In the [original issue](https://github.com/rancher/dashboard/issues/5460) is requested to keep the preference saved in the user preferences.

These can be retrieved with `this.$store.getters['prefs/get'](PUT_YOUR_CONSTANT_HERE)`, while the value has to be set in the [prefs file](https://github.com/rancher/dashboard/blob/master/store/prefs.js)